src/os_esp32.cc

```diff
@@ -741,19 +741,20 @@ void OS::heap_summary_report(int max_pages, const char* marker) { }
 #endif // def TOIT_CMPCTMALLOC
 
 static const int TOIT_IMAGE_DATA_SIZE = 1024;
-static const int TOIT_CONFIG_IMAGE_SIZE = TOIT_IMAGE_DATA_SIZE - UUID_SIZE;
+static const int TOIT_CONFIG_IMAGE_SIZE = TOIT_IMAGE_DATA_SIZE - UUID_SIZE - sizeof(uint32);
```

Comment:
Make that a constant.
Also explain why they are removed. (I'm guessing we reserve this space).